### ISSUE TITLE
Change "Offer to save logins" behaviour

### ISSUE THREAD

**Issue by StFIT** [mozilla/policy-templates#193]
NOW:   If this policy is enabled or not configured, Firefox will offer to save website logins and passwords. If this policy is disabled, firefox will not offer to save website logins and passwords.
(the option to save passwords is greyed out, no more password saving is possible)

COOL SOLUTION:  new behavoir of "not configured"

Not configured:    user preference to save password is not enabled / users can enable to save website logins and passwords themself
activated:              firefox save website logins and passwords by default (like now)
deactivated:          the option to save passwords is greyed out, no more password saving is possible (like now)

***********
Request add a new policy: 

A new policy would/could be very perfect for us:  force using master password if the user manually enables saving logins and password.

**Comment by mkaply** [comment]
Not configured means "the default" so in this case, it would be that Firefox saves logins and passwords.

As far as master password goes, there is no easy way to force a user to use the master password.

**Comment by StFIT** [comment]
Correct. That's not what we want. We want the user's setting to save passwords not to be set, AND at the same time not completely disabled / grayed out. The user should be able to switch on saving of logins and passwords.

**Comment by StFIT** [comment]
Again, to make it clear. The policy should not "lock" the preference.

**Comment by StFIT** [comment]
Is it possible to change this GPO behaviour so it works as if you put
defaultPref("signon.rememberSignons", false)
in the mozilla.cfg?

**Comment by LucvandenBerg** [comment]
Hi Mike, any updates on this? We were used to not offering to save the login by default but the ability for users to enable this. Now when this policy is disabled the option to save passwords is locked (greyed out).

**Comment by mkaply** [comment]
Unfortunately I can't change the existing behavior because I don't want to break existing users.

I think I'm going to add a new policy called OfferToSaveLoginsDefault that just sets the default value as you suggested?

How does that sound?

Or should it be DefaultOfferToSaveLogins

**Comment by LucvandenBerg** [comment]
That sounds great, Mike. That would solve our problem. I noticed both (..)Default and Default(..) are being used. OfferToSaveLoginsDefault sounds more logical to me. Thanks.

**Comment by mkaply** [comment]
Opened https://bugzilla.mozilla.org/show_bug.cgi?id=1573967

**Comment by StFIT** [comment]
We use as a workaround mozilla.cfg with

defaultPref("signon.rememberSignons", false);

that does the job for us. The Option false is default and not grayed out so users can change.
Better would be a solution within GPO.

I also would prefer OfferToSaveLoginsDefault :)
